### PR TITLE
now with eslint

### PR DESCRIPTION
**Pull request by Dmitriy-Pashko**

### CODE REVIEW

File: src/view/WorkerApp.js [hunk 1]
```diff
@@ -1,84 +1,86 @@
...
-        let workers = this.state.data;
-        return axios.post(this.props.url, worker).then((data)=>{
-            let newWorkers = workers.concat([data]);
-            this.setState({ data: newWorkers });
-            return data;
-        })
-        .catch(err => {
-        console.error(err);
-        this.setState({ data: workers });
+      const workers = this.state.data;
```
Comment (warenekit) [inline src/view/WorkerApp.js]: needs one line of space between const declaration and function declarations

File: src/view/WorkerApp.js [hunk 2]
```diff
@@ -1,84 +1,86 @@
...
+      return axios.post(this.props.url, worker).then((data) => {
+        const newWorkers = workers.concat([data]);
```
Comment (warenekit) [inline src/view/WorkerApp.js]: space

File: src/view/WorkerForm.js [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
     }
-    render(){
-        return(    
-        <div className="modal fade" id="exampleModal" tabIndex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+    this.props.onWorkerSubmit({
+      name, bdate, position, salary,
+    }).then(() => {
+      this.setState({
+        name: '', bdate: '', position: '', salary: '',
+      });
```
Comment (warenekit) [inline src/view/WorkerForm.js]: needs a return

File: src/view/WorkerApp.js [hunk 4]
```diff
@@ -1,84 +1,86 @@
...
+    super(props);
+    this.state = { data: [] };
+    // this.WAppController=WAppController(this);
+  }
+  
+  componentDidMount() {
+    this.loadWorkersFromServer();
+  }
+  loadWorkersFromServer = () => {
+    axios.get(this.props.url)
```
Comment (warenekit) [inline src/view/WorkerApp.js]: needs a return axios

File: src/view/WorkerApp.js (hunk 1 above)
Comment (Dmitriy-Pashko) [reply src/view/WorkerApp.js]: trailing spases not allowed

Comment (Dmitriy-Pashko) [reply src/view/WorkerApp.js]: I can delete rule cuse it is stupid

Comment (Dmitriy-Pashko) [reply src/view/WorkerApp.js]: but whatever, it dissapered

File: src/view/WorkerApp.js (hunk 4 above)
Comment (Dmitriy-Pashko) [reply src/view/WorkerApp.js]: changes was made

**Pull request merged**